scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -140,16 +140,17 @@ _deploy_deb()
 }
 
 _install_user_mode_redhat()
-{   
-    CMDS=("dnf install --refresh -y intel-igc-opencl-1.0.9441-i643.el8.x86_64 \
-           intel-media-21.4.1-i643.el8.x86_64 \
-           level-zero-1.6.2-i643.el8.x86_64 \
-           intel-level-zero-gpu-1.2.21786-i643.el8.x86_64 \
-           intel-opencl-21.49.21786-i643.el8.x86_64 \
-           intel-igc-core-1.0.9441-i643.el8.x86_64 \
-           intel-ocloc-21.49.21786-i643.el8.x86_64 \
-           ocl-icd-2.2.12-1.el8.x86_64 \
-           intel-gmmlib-21.3.3-i643.el8.x86_64")
+{  
+    CMDS=("rpm -ivh https://vault.centos.org/centos/8/AppStream/x86_64/os/Packages/mesa-filesystem-21.1.5-1.el8.x86_64.rpm" \
+          "dnf install --refresh -y \
+            intel-igc-opencl-1.0.10409-i699.3.el8.x86_64 \
+            intel-media-22.2.2-i699.3.el8.x86_64 \
+            level-zero-1.7.9-i699.3.el8.x86_64 \
+            intel-opencl-22.10.22597-i699.3.el8.x86_64  \
+            intel-igc-core-1.0.10409-i699.3.el8.x86_64 \
+            intel-ocloc-22.10.22597-i699.3.el8.x86_64 \
+            intel-gmmlib-22.0.3-i699.3.el8.x86_64" \
+          "rpm -ivh http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/ocl-icd-2.2.12-1.el8.x86_64.rpm" )	
```

Comment:
@vladimir-paramuzov are we OK to change driver version here? have we validated with this one?
what is the process of changing driver versions?

Reply:
There was performed a range of functional tests and benchmarking on available gpu platforms. Performance is about the same. The reason in this update is to have a version from the official repository for RH8.5 version